app/registries/foreman/settings/auth.rb

```diff
@@ -1,3 +1,9 @@
+HASHING_ALGORITHMS = {
+  "sha1" => _("SHA1"),
+  "bcrypt" => _("BCrypt"),
+  "pbkdf2sha1" => _("PBKDF2 SHA1"),
+}
+
```

Comment:
This is incorrect, the translations will be evaluated on code load.
Translated strings should be inside of the proc, that is why we have it in the first place.
Why are we translating the strings anyway? and could we not just define this inplace instead of having global constant?